pages/blacklisting.rst

```diff
@@ -1,46 +1,9 @@
+:orphan:
+
 .. _blacklisting:
 
 ************
 Blacklisting
 ************
 
 .. note:: Since Graylog 2.0 you can use the :doc:`processing pipelines <pipelines>` for blacklisting.
```

Comment:
Is there a black listing example on the processing pipeline page that we can link to?

Reply:
currently not - but I'll add one and then link to that